cherryontech/unmatched
# adds gradient to image

## Description



## Related Tickets & Documents


## Acceptance Criteria



## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
|     | :sparkles: New feature     |
|     | 🎨 Style                   |
|     | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|  ✓   | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Mobile & Desktop Screenshots/Recordings


<img width="1509" alt="Screenshot 2023-11-13 at 2 30 59 PM" src="https://github.com/cherryontech/unmatched/assets/110904882/bcb35d80-ef6f-4f66-86bd-62df98c15a23">

## What I learned
Figma provides some style hints for tailwind. Background color can be used for double border
## Testing steps

Switch to this branch with the following command: git switch feature/UM-14-biopage-fix
Install dependencies with the following command: npm i
Run the following scripts: npm run dev to start Vite's dev build & npm run tailwind to start the Tailwind CSS build
View the app in browser (the npm run dev script will return the url you can see the app at)
Navigate to the the Resources page and click on the learn more button of one of the cards .
see newly added gradient ring around bio image  

## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
|     | 🙅 no documentation needed    |


## What gif best describes this PR or how it makes you feel?
